transport_nantes/transport_nantes/settings.py

```diff
@@ -202,6 +202,12 @@
             # 'when': 'd',
             # 'utc': True,
             # 'backupCount': 10,
+        },
+        'rgeocoding_handler': {
+            'level': 'INFO',
+            'class': 'logging.FileHandler',
+            'filename': settings_local.LOG_DIR + "rgeocoding.log",
```

Comment:
Let's not make a separate log for every GCP service.  Perhaps call it tn_gcp.log?